1. Contributing/Reporting Issues/README.md

```diff
@@ -2,29 +2,52 @@
 
 You don't have to write code to contribute! We are always looking for help with creating or updating documentation, testing our apps (web, desktop and mobile) and reporting bugs.
 
-We appreciate any & all bug reports! Did you find a bug? Reporting is easy!
+## Be sure its a bug
+We are striving to make our issues a list of actionable items for the team.  So we want to keep it a list of issues and feature requests.
 
-Simply open an [Issue](https://github.com/RocketChat/Rocket.Chat/issues) in the applicable repository.
+Examples of things that aren't considered bugs or feature requests:
+* Can't figure out how to do something
+* Having issues accessing your server
+* Can't get the server installed
 
-Here are some suggested things to include, when applicable, which may help us troubleshoot a problem:
+If your issue isn't a bug please see [Getting Support](/2. Getting%20Support)
+
+## Reporting a bug
+
+Did you find a bug? Reporting is easy!
+
+1. [Search the Issues](https://github.com/RocketChat/Rocket.Chat/issues) for your bug.  If you find it, give it a thumbsup and add some additional information to the ticket if differing.  This will help give us a fuller picture.
+2. Make sure you are running the latest version of Rocket.Chat. Its very likely simply updating will resolve your issue.
+3. Try to narrow it down to reproducable steps.  If we can reproduce it, we will be able to fix the issue a lot quicker.
+4. Look at logs for errors. See below for how to gather logs.
+5. If you got this far then: [Open an issue](https://github.com/RocketChat/Rocket.Chat/issues/new)
+
+### Things to Include in Bug Report
+
+Here are some things to include, when applicable, which may help us troubleshoot a problem:
 
 * Operating System / Version / Architecture (64 bit?)
 * Browser type & Version (if web app), any browser add-ons which may be involved (e.g. AdBlocker, NoScript, etc.)
-* Desktop Environment (if desktop app) / Version
-* Rocket.Chat edition / Version or [build number](/2. Getting%20Support)
+* Rocket.Chat Version
 * Expected behavior vs. Actual behavior (In other words, the "bug")
 * Can it be reproduced? If yes, how?
```

Comment:
I think we partially capture that by asking if reproducible.  I'll adjust.  

I think important thing here is we want them to realize for us to fix it.. we need to be able to reproduce the error